internal/util/metrics/metrics.go

```diff
@@ -62,13 +83,76 @@ func Init() {
 		[]string{"origin", "origin_type", "method", "status", "http_status", "path"},
 	)
 
+	CacheObjectOperations = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "trickster_cache_operations_objects",
+			Help: "Count of operations performed on a Trickster cache.",
+		},
+		[]string{"cache", "type", "operation", "status"},
```

Comment:
Let's avoid `type` as a label, it's a bit generic and can cause conflicts with discovery. How about `cache_name`, and `cache_type` as the first two labels on these metrics.